source/NVDAObjects/UIA/edge.py

```diff
@@ -449,22 +452,26 @@ def _get_isCurrent(self):
 		return False
 
 	def _get_placeholder(self):
-		ariaProperties=splitUIAElementAttribs(self.UIAElement.currentAriaProperties)
-		ariaPlaceholder = ariaProperties.get('placeholder', None)
+		ariaPlaceholder = self.ariaProperties.get('placeholder', None)
 		return ariaPlaceholder
 
 	def _get__isTextEmpty(self):
-		ti = self.makeTextInfo(textInfos.POSITION_FIRST)
-		ti.move(textInfos.UNIT_CHARACTER, 1, endPoint="end")
-		# we were unable to expand the range, it is collapsed
-		if ti.isCollapsed:
-			return True
 		# NOTE: we can not check the result of the EdgeTextInfo move implementation to determine if we added
 		# any characters to the range, since it seems to return 1 even when the text property has not changed.
-		ti.move(textInfos.UNIT_CHARACTER, 1, endPoint="end")
-		text = ti.text
-		# Edge can report a range of values for empty fields:
-		if text == " " or text == "\n" or text =="\n " or text=="\n\n" or text ==" \n":
+		# Also we can not move (repeatedly by one character) since this can overrun the end of the field in edge.
+		# So instead, we use self to make a text info (which should have the right range) and then use the edge
+		# specific _rangeObj.getText function to get a subset of the full range of characters.

```

Comment:
I'd suggest changing this from "Edge specific" to "UIA specific", since that method is a UIA method.